package.json

```
@@ -2,20 +2,22 @@
   "name": "@axa-fr/design-system",
   "version": "0.1.0",
   "description": "",
+  "packageManager": "npm@11.0.0",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/AxaFrance/design-system.git"
   },
   "scripts": {
-    "start": "concurrently \"npm run start:*\"",
-    "start:slash": "concurrently \"npm run start -w=slash/css\" \"npm run start:storybook -w=slash/css\" \"npm run start:storybook -w=slash/react\" -c auto -n css,css:storybook,react:storybook",
-    "start:look-and-feel": "concurrently \"npm run start -w=client/look-and-feel/css\" \"npm run start:storybook -w=client/look-and-feel/css\" \"npm run start:storybook -w=client/look-and-feel/react\" -c auto -n css,css:storybook,react:storybook",
-    "build": "concurrently \"npm run build:*\"",
-    "build:slash": "npm run build --workspace=slash && npm run build:storybook --workspace=slash",
-    "build:look-and-feel": "npm run build --workspace=client/look-and-feel && npm run build:storybook --workspace=client/look-and-feel",
-    "lint": "concurrently \"npm run lint:*\"",
-    "lint:slash": "concurrently \"npm:eslint --workspace=slash\" \"npm:stylelint -w=slash/css\" \"npm:prettier --workspace=slash\" \"npm:tsc --workspace=slash\" -c auto -n eslint,stylelint,prettier,tsc",
-    "lint:look-and-feel": "concurrently \"npm:eslint --workspace=client/look-and-feel\" \"npm:stylelint -w=client/look-and-feel/css\" \"npm:prettier --workspace=client/look-and-feel\" \"npm:tsc --workspace=client/look-and-feel\" -c auto -n eslint,stylelint,prettier,tsc",
+    "dev": "turbo dev --filter=!./samples/vite",
+    "dev-lf": "turbo dev --filter=@axa-fr/stories-look-and-feel-css --filter=@axa-fr/stories-look-and-feel",
+    "dev-slash": "turbo dev --filter=@axa-fr/slash-stories-css --filter=@axa-fr/slash-stories",
```

Pour plus de cohérence avec les autre script
```suggestion
    "dev:lf": "turbo dev --filter=@axa-fr/stories-look-and-feel-css --filter=@axa-fr/stories-look-and-feel",
    "dev:slash": "turbo dev --filter=@axa-fr/slash-stories-css --filter=@axa-fr/slash-stories",
```

---

done